l10n_nl_tax_statement/models/l10n_nl_vat_statement.py

```diff
@@ -173,7 +173,16 @@ def _get_company_ids_full_list(self):
         string="Unreported Journal Entries",
         compute='_compute_unreported_move_ids'
     )
-    unreported_move_from_date = fields.Date()
+
+    @api.model
+    def _default_unreported_move_from_date(self, date=None):
+        """ Prevent legacy move lines from being selected as unreported by
+        defaulting the unreported date to the start date of the earliest
+        statement.
+        """
```

Comment:
Thanks for this PR!
Is this method missing the body?

Reply:
huh yes! Let me try to find it back...

Reply:
Remnant of an earlier refactoring. Removed.